openshift-install-powervs

```diff
@@ -851,7 +851,13 @@ function variables_nodes {
     question "Select the flavor for the cluster nodes:" "${flavor_list[*]}" yes 
     if [ "${value}" != "CUSTOM" ]; then
       cat automation/compute-vars/$value.tfvars >> "$VAR_TEMPLATE"
-      if [ "${no_of_bastion}" -eq 2 ]; then awk -i inplace '{ if ( $1 == "bastion" ) $12 = 2 ; print $0 }' $VAR_TEMPLATE; fi
+      if [ "${no_of_bastion}" -eq 2 ]; then
+        if [[ "$OS" == "darwin" ]]; then
+          sed -i '' -e 's/\(bastion.*\) 1/\1 2/' $VAR_TEMPLATE;
```

Comment:
Is this to handle non-GNU sed in OSx? 
Does the issue occur if you install gnu sed, make it default in OSX and run the script?

Reply:
@bpradipt I spoke to @mgiessing and the `gnu-sed` approach isn't working for him, we are good to merge this PR for now.

Reply:
This will fail if gnu sed is used and is default for OSX.
You'll need to check for both darwin and non-GNU sed.